rattan-core/src/cells/mod.rs

```diff
@@ -256,6 +313,115 @@ impl Packet for StdPacket {
     }
 }
 
+/// A wrapper around a packet structure to analyse a cell's behaviour.
+///
+/// It stores the packet's creation timestamp, allowing us to inspect how long the packet
+/// has been created, in terms of logical timestamp.
+/// This exists only for test code. Especially for the test of cells that may impose
```

Comment:
Add a newline between the comments. Make them into two paragraphs in the doc comment.